assets/hydras/amiko_hydra.py

```diff
@@ -0,0 +1,265 @@
+"""
+Small script meant for use in an interactive shell,
+which does the evolution of Amiko hydras.
```

Comment:
+1 for leaving comments in your code! 😄 Consider also including a comment with author, date, and license information too.

Reply:
added the MIT license text in most recent commit